docs/fleet/provision.md

```diff
@@ -1,278 +1,210 @@
 
-  {{< alert title="Alert" color="note" >}}
-  When you stop the Viam Agent, the agent will stop `viam-server` as well.
-  {{< /alert >}}
-
-  ```sh {class="command-line" data-prompt="$"}
-  sudo systemctl stop viam-agent
-  ```
-
-- To restart the Viam Agent:
-
-  {{< alert title="Alert" color="note" >}}
-  When you restart the Viam Agent, the agent will restart `viam-server` as well.
-  {{< /alert >}}
-
-  ```sh {class="command-line" data-prompt="$"}
-  sudo systemctl restart viam-agent
-  ```
-
-- To completely uninstall the Viam Agent and `viam-server`, run the following command:
+If you are not using Flutter or TypeScript and would like to use provisioning, please [contact us](mailto:support@viam.com).
 
-  ```sh {class="command-line" data-prompt="$"}
-  sudo /bin/sh -c "$(curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/uninstall.sh)"
-  ```
+Regardless whether provisioning happens through the captive web portal or a mobile app, the process follows the same pattern:
 
-  This command uninstalls Viam Agent, `viam-server`, the machine configuration file (<file>/etc/viam.json</file>), and the provisioning configuration file (<file>/etc/viam-provisioning.json</file>).
+{{<video webm_src="/platform/provisioning-demo.webm" mp4_src="/platform/provisioning-demo.mp4" alt="Using the Viam mobile app to provision a new machine with viam-agent." poster="/platform/provisioning-demo.jpg" class="alignright" max-width="400px" style="margin-left: 2rem">}}
 
-  For more information, see [Viam Agent management](https://github.com/viamrobotics/agent#management).
+1. When a machine that has `viam-agent` installed and `agent-provisioning` configured comes online, `agent-provisioning` creates a WiFi hotspot.
 
-### View Viam Agent logs
+   - The [`agent-provisioning` configuration](#configuration) is at <file>/etc/viam-provisioning.json</file.
+   - The [`viam-agent` configuration](/configure/agent/#configuration), if provided, is at <file>/etc/viam.json</file> alongside the rest of the machine configuration, including which fragments to use (if any).
 
-The Viam Agent writes log messages to the [Viam app](https://app.viam.com/).
-You can find these messages on the [**LOGS** tab](/cloud/machines/#logs) of your machine's page.
+1. The end user uses their mobile device or computer and connects to the WiFi hotspot.
 
-Viam Agent only sends messages when your machine is online and connected to the internet.
-If your machine is offline, log messages are queued, and are sent to the Viam app once your machine reconnects to the internet.
+   - By default, the hotspot network is named `viam-setup-HOSTNAME`, where `HOSTNAME` is replaced with the hostname of your machine.
+     The WiFi password for this hotspot network is `viamsetup` by default.
+     You can customize these values in the machine's or fragment's [`viam-agent` configuration](/configure/agent/#configuration).
```

Comment:
These are mostly meant to be customized in the provisioning configuration (/etc/viam-provisioning.json), not the Cloud configuration, as only that can be read/used when offline (e.g. before the user has first provisioned the machine.)

Reply:
that makes sense but why is it configurable in the agent config as well then?

Reply:
maybe it isn't

Reply:
Can you check the attributes in https://docs-test.viam.dev/3243/configure/agent/#agent-provisioning